### ISSUE TITLE
Option to always show which_key

### ISSUE THREAD

**Issue by lo1tuma** [nvim-telescope/telescope.nvim#2752]
Currently displaying `which_key` is only possible to be triggered via a keymap. It would be nice to have an option that `which_key` is always displayed as long as the telescope prompt is open and it automatically disappears when the prompt gets closed.

**Comment by smackesey** [comment]
A related and perhaps prerequisite request is the ability to keep `which_key` open until the Telescope prompt closes. Currently you can set `close_with_action = false`, which will keep `which_key` open while you scroll through the list, but it stays open even after you close Telescope, which is quite annoying and seems like a bug.

**Comment by jamestrew** [comment]
> A related and perhaps prerequisite request is the ability to keep `which_key` open until the Telescope prompt closes. Currently you can set `close_with_action = false`, which will keep `which_key` open while you scroll through the list, but it stays open even after you close Telescope, which is quite annoying and seems like a bug.

merged https://github.com/nvim-telescope/telescope.nvim/pull/2819 to fix this

**Comment by jamestrew** [comment]
I'm not opposed to adding an option to always start telescope with which_key showing but I'm slightly hesitant since by default with most people neovim window size imagine, the which_key window and main telescope windows will overlap. 
This depends on the theme, config and such as well but I think generally by default, I'm not going to be particularly elegant.

**Comment by smackesey** [comment]
>I'm not opposed to adding an option to always start telescope with which_key showing but I'm slightly hesitant since by default with most people neovim window size imagine, the which_key window and main telescope windows will overlap. This depends on the theme, config and such as well but I think generally by default, I'm not going to be particularly elegant.

This is a fair point but IMO if the `which_key` window is good enough to show at all, it's good enough to have an option to show by default. Also of course this option can be off by default. I think the functionality gains for those who want this feature outweigh the other concerns.

I'd be happy to put up a PR following up #2819 to implement it.

**Comment by jamestrew** [comment]
Yeah let's go for it. I suppose we can clean up the overlapping windows later if it becomes an issue.
